### PR TITLE
meson: conditionalize systemd unit file installation

### PR DESCRIPTION
**Pull request by Johnnynator**
libratbag works fine also on systems that use elogind instead of full blown systemd.

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -437,35 +437,39 @@ configure_file(input : 'dbus/org.freedesktop.ratbag_devel1.conf.in',
...
+			'''.format(default_unitdir))
+			unitdir = default_unitdir
+		else
+			unitdir = intended_unitdir
+		endif
 	endif
 endif
 
 config_bindir = configuration_data()
 config_bindir.set('bindir', join_paths(get_option('prefix'), get_option('bindir')))
```
Comment (whot) [inline meson.build]: are we using config_bindir anywhere else? If not, we can move it into the if condition, then you don't need to check for `dep_systemd.found()` twice.

Comment (Johnnynator) [reply meson.build]: True. Has to be fixed in master then.

**Comment by Johnnynator** [comment]
Closed since #510 got merges

File: meson.build (hunk 1 above)
Comment (whot) [reply meson.build]: nah, doesn't need to be fixed, just checked and it's used for the dbus service file too

**Pull request closed without merging**